engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -64,38 +44,26 @@ object MetadataServiceActor {
 
 case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config) extends Actor {
 
-  private var journal = List.empty[MetadataEvent]
+  private val dataAccess = DataAccess.globalDataAccess
```

Comment:
Maybe this should be injected via the constructor? Might facilitate a little better testing.